client/src/components/CreatePost/StyledPostAs.js

```diff
@@ -36,7 +36,7 @@ const Container = styled(Modal)`
   .ant-modal-body {
     display: flex;
     max-height: 49.6rem;
-    overflow-y: scroll;
+    overflow-y: auto;
```

Comment:
I think this was probably added to prevent scrollbars?
Maybe the property can just be removed? 
Did you also try this with different things like if the content grows and things like that?

Reply:
Yes. I tried with smaller screen sizes and works fine for width larger than 120px with **auto** and also **without** the property.
Should I go with removing the overflow-y?

Reply:
yeah just removing is better if it's not needed, less code is usually better :)